content/source/docs/cloud/api/variable-sets.html.md

```diff
@@ -0,0 +1,613 @@
+---
+layout: "cloud"
+page_title: "Variable Sets - API Docs - Terraform Cloud and Terraform Enterprise"
+---
+
+# Variable Sets API
+
+A[variable set](/docs/cloud/workspaces/variables.html#variable-sets) is a resource that allows you to reuse the same variables across multiple workspaces. For example, you could define a variable set of provider credentials and automatically apply it to one or all workspaces.
+
+You need [`read variables` permission](/docs/cloud/users-teams-organizations/permissions.html#general-workspace-permissions) to view the variables for a particular workspace and to view the variable sets in the owning organization. You need [read and write variables permissions](/docs/cloud/users-teams-organizations/permissions.html#general-workspace-permissions) to create and edit variable sets. 
+
+
+## Create a Variable Set
+
+`POST organizations/:organization_name/varsets`
+
+| Parameter            | Description                                                                                                                                                              |
+| -------------------- | ----------------------------------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization the workspace belongs to.                                                |
+
+### Request Body
+
+Properties without a default value are required.
+
+| Key path                               | Type           | Default | Description
+-----------------------------------------|----------------|---------|------------
+`data.name`                              | string         |         | The name of the Variable set
```

Comment:
```suggestion
`data.name`                              | string         |         | The name of the variable set
```

Reply:
are there any length constraints here that we should tell folks about? Or disallowed characters?